aspnetcore/fundamentals/middleware/sample/Culture/RequestCultureMiddleware.cs

```diff
@@ -0,0 +1,32 @@
+﻿using Microsoft.AspNetCore.Http;
+using System.Globalization;
+using System.Threading.Tasks;
+
+namespace Culture
+{
+    public class RequestCultureMiddleware
+    {
+        private readonly RequestDelegate next;
```

Comment:
We usually prefix these variables with _ so `_next` would be recommended here. With this rename, you can also remove the `this.` when accessing or using this field.